src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -663,29 +659,60 @@ func (c controller) validateConsoleAvailability() bool {
 	}
 
 	c.log.Info("Console operator is available")
-	return true
+
+	operatorStatusInService, err := c.ic.GetClusterMonitoredOperator(utils.GenerateRequestContext(), c.ClusterID, consoleOperatorName)
+	if err != nil {
+		c.log.WithError(err).Errorf("Failed to get cluster %s console operator status", c.ClusterID)
+		return false
+	}
+
+	if operatorStatusInService.Status == models.OperatorStatusAvailable {
+		c.log.Infof("Service acknowledged console operator is available for cluster %s", c.ClusterID)
+		return true
+	}
+
+	return false
 }
 
 func (c controller) waitingForClusterVersion() error {
```

Comment:
similar comment to the comment I made about `validateConsoleAvailability`

Reply:
It would require changing the behavior of the CVO reporting. I will do that as part of MGMT-5188 where I will merge the same usage of the CVO reporting with the console.

Reply:
Since renaming to `waitingForClusterVersionAndAcknowledgeInAssistedService` is not an option, let's just add a short docstring :)

Reply:
You got it. :) 
```
// waitingForClusterVersion checks the Cluster Version Operator availability in the
// new OCP cluster. A success would be announced only when the service acknowledges
// the CVO availability, in order to avoid unsycned scenarios.
//
// This function would be aligned with the console operator reporting workflow
// as part of the deprecation of the old API in MGMT-5188.
```